main.py

```diff
@@ -62,3 +55,43 @@ def chat_Future(future):
 
   #     return random.choice(responses)
 
+# handle_input returns a bool indicating if the input
+# from the user was able to be routed to a different method successfully
+# True = success
+# False = failed
+def handle_input(user_input):
+  if user_input == 'food':
+    chat_Food()
+  elif user_input == 'activities':
+    chat_Activities()
+  elif user_input == 'music':
+    chat_Music()
+  elif user_input == 'future':
+    chat_Future()
+  elif user_input == quit_action:
+    # https://docs.python.org/3/reference/simple_stmts.html#the-pass-statement
+    pass
```

Comment:
`pass` basically just means "do nothing" in Python. I added that check here because I didn't want it to log the message "I'm sorry, I didn't understand. Let's try again." if we're processing the user's request to quit the program!